src/main/groovy/de/dkfz/roddy/tools/versions/Version.groovy

```diff
@@ -0,0 +1,166 @@
+package de.dkfz.roddy.tools.versions
+
+import groovy.transform.CompileDynamic
+import groovy.transform.CompileStatic
+
+import java.text.ParseException
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+class VersionInterval {
+    final Version from
+    final Version to
+
+    def contains(Version query) {
+        return query >= from && query <= to
+    }
+}
+
+
+@CompileStatic
+class Version implements Comparable<Version> {
```

Comment:
I see, that this class is very specific for Roddy. Maybe you could reflect this in the class name or put it in a comment. For us it is very clear but others might get confused.

Some examples in the comment could also help

Reply:
We have not yet clarified the versioning scheme for RoddyToolLib and BatchEuphoria. If this class is really Roddy-specific, I suggest moving it there.